File: CodigosJava/DespachoComidas/ComidaSimple.java
package DespachoComidas;

import java.sql.Time;

public class ComidaSimple extends Comida {
    private int calorias;
    private int precio;
    private Time tPreparacion;

    public ComidaSimple(String nombre, String tipo, String modoPreparacion, int calorias,
                        int precio, Time tPreparacion){
        super(nombre, tipo, modoPreparacion);
        this.calorias = calorias;
        this.precio = precio;
        this.tPreparacion = tPreparacion;
    }
    @Override
    public int getCalorias() {
        return this.calorias;
    }
    public void setCalorias(int calorias) {
        this.calorias = calorias;
    }
    @Override
    public int getPrecio() {
        return this.precio;
    }
    public void setPrecio(int precio) {
        this.precio = precio;
    }
    @Override
    public Time getPreparacion() {
        return this.tPreparacion;
    }
    public void setPreparacion(Time tPreparacion) {
        this.tPreparacion = tPreparacion;
    }
}

File: codigoPython/DespachoComidas/ComidaSimple.py
from Comida import Comida


class ComidaSimple(Comida):

    def __init__(self, nombre, tipo, modoPreparacion, aceptar, caloria, precio, tPreparacion):  # constructor
        super().__init__(nombre, tipo, modoPreparacion, aceptar)
        self.__caloria = caloria
        self.__precio = precio
        self.__tPreparacion = tPreparacion

    def get_precio(self):
        return self.__precio

    def get_calorias(self):
        return self.__caloria

    def get_preparacion(self):
        return self.__tPreparacion
